pkg/validate/apparmor_linux.go

```diff
@@ -82,37 +82,64 @@ var _ = framework.KubeDescribe("AppArmor", func() {
 			})
 
 			It("should fail with an unloaded profile", func() {
-				profile := apparmorProfileNamePrefix + "non-existent-profile"
+				profile := &runtimeapi.LinuxContainerSecurityContext{
+					ApparmorProfile: apparmorProfileNamePrefix + "non-existent-profile",
+				}
```

Comment:
The question is: Do we want to preserve the old tests until the field is gone? Make them optional?

Reply:
old test cases are preserved here. I think the right thing to do here is to rename test cases indicating that those are old fields